RatTracker-WPF/RatTracker-WPF/CmdrJournalParser.cs

```diff
@@ -205,16 +203,32 @@ private void CmdrLogMonitor(object sender, EventArgs args)
 
             if (_newFile)
             {
-                var existingLines = File.ReadAllLines(fi.FullName);
+                string[] existingLines = {};
+                try
+                {
+                    existingLines = File.ReadAllLines(fi.FullName);
+                }
+                catch (Exception e)
```

Comment:
this still won't fix the fact that xlex is erroring on read... I would love to find out why.